Symfony/CS/Tokenizer/TokensAnalyzer.php

```diff
@@ -211,7 +211,79 @@ public function isArrayMultiLine($index)
     }
 
     /**
-     * Check if there is a lambda function under given index.
+     * Returns the attributes of the method under the given index.
+     *
+     * @param int $index Token index of the method (T_FUNCTION)
+     *
+     * @return array
```

Comment:
Not nothing, but maybe not enough.
Should I update the return type to something like described here?
https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#7-describing-hashes